frontend/src/components/ui/DifficultyButton.js

```diff
@@ -0,0 +1,43 @@
+import { Button } from "@mui/material";
+
+import React from "react";
+
+import styles from "./DifficultButton.module.css";
+
+function DifficultyButton(props) {
+  //Logic for button color
+  let btnStyling = styles.btn;
+
+  switch (props.buttonText) {
+    case "Easy":
+      btnStyling += " " + styles.btn_easy;
+      break;
+    case "Medium":
+      btnStyling += " " + styles.btn_medium;
+      break;
+    case "Hard":
+      btnStyling += " " + styles.btn_hard;
+      break;
+    default:
+      break;
+  }
+
+  return (
+    <Button
+      className={btnStyling}
+      variant="outlined"
+      size="large"
+      onClick={chooseDifficulty}
+      id={props.buttonText}
+    >
+      {props.buttonText}
+    </Button>
+  );
+}
+
+//Clicking on Difficulty button
+function chooseDifficulty(e) {
```

Comment:
Sorry but can you help moving this function into the functional component `DifficultyButton` thanks thanks. Because I expect that we need to use hook to do some route navigation.

Also, just a nitpick, after you put this function into `DifficultyButton`, can you help write it in the form of:
`const <function_name> = (<parameters here>) => { // Logic here}`

Just for consistency, and it's easier to differentiate between functional component and function 😄 Thanks!

Reply:
Right that's true. So we will do some navigation in this function.